src/rpm_ostree/cli_status.rs

```diff
@@ -0,0 +1,59 @@
+//! Interface to `rpm-ostree status --json`.
+
+#![allow(unused)]
+
+use super::Release;
+use failure::{bail, ensure, format_err, Fallible, ResultExt};
+use serde::Deserialize;
+
+/// JSON output from `rpm-ostree status --json`
+#[derive(Debug, Deserialize)]
+pub struct StatusJSON {
+    deployments: Vec<DeploymentJSON>,
+}
+
+/// Partial deployment object (only fields relevant to zincati).
+#[derive(Debug, Deserialize)]
+pub struct DeploymentJSON {
+    booted: bool,
+    checksum: String,
```

Comment:
Great catch, I was vaguely aware of this nuance but I didn't realized it would be relevant here. I'm fine with keeping the current behavior of rpm-ostree.

For reference, this specific codepath is not currently used. But I think future logic will depend on introspection, so I added this preemptively.